mm/2s2h/Enhancements/Trackers/ItemTracker/ItemTrackerSettings.cpp

```diff
@@ -27,7 +27,7 @@ std::vector<std::string> listOrder = {
 };
 
 std::vector<std::string> randoListOrder = {
-    "Frogs", "Boss Souls", "Owl Statues", "Tingle Maps", "Misc",
+    "Frogs", "Boss Souls", "Owl Statues", "Tingle Maps", "Clocks", "Misc",
```

Comment:
This is user-facing, so for consistency let's make this "Time"

```suggestion
    "Frogs", "Boss Souls", "Owl Statues", "Tingle Maps", "Time", "Misc",
```